libpod/oci_linux.go

```diff
@@ -246,7 +246,9 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string, res
 	}
 
 	logDriver := KubernetesLogging
-	if ctr.LogDriver() != "" {
+	if ctr.LogDriver() == JSONLogging {
+		logrus.Errorf("json-file logging specified but not supported. Choosing k8s-file logging instead")
```

Comment:
This should be a Warnf not an Errorf.

Reply:
I really want to keep the Errrof here - we need to make it very obvious that they should switch to k8s-file

Reply:
I would be fine either way, I'll let you two decide. I will weigh in that the only reason a user cares about the difference is if they want to read the log as a json. If we leave it as a warning, and document it, we can point users to the fact they can't read it as json, and thus not have a loud error.